src/components/Patient/Utils.ts

```diff
@@ -17,3 +17,16 @@ export function isPatientMandatoryDataFilled(patient: PatientModel) {
     patient.blood_group
   );
 }
+export const isPatientDischarged = (
+  patientData: PatientModel,
+  facilityId: string | undefined,
+) => {
+  if (!patientData.is_active) {
+    return true;
+  }
+  const patientFacilityId = patientData.facility_object?.id;
+  if (!patientFacilityId || !facilityId) {
+    return false;
+  }
+  return patientFacilityId.toString() !== facilityId;
+};
```

Comment:
we should also be checking if the last_consultation.discharge_date has a value or not. This is because what if patient is readmitted to the same facility consecutively and readmission is not having a consultation filed yet